packages/client/wallets/smart-wallet/src/blockchain/wallets/passkey.ts

```diff
@@ -102,6 +106,32 @@ export class PasskeyAccountService {
             Authorization: `Bearer ${user.jwt}`,
         };
     }
+
+    private mapError(error: any, passkeyName: string) {
+        if (error.code === "ERROR_PASSTHROUGH_SEE_CAUSE_PROPERTY" && error.name === "NotAllowedError") {
```

Comment:
this isn't a var in the zerodev SDK anywhere we'd be able to point to is it?
would be nicer if there were a `import { PASSKEY_ERROR_CODE } from "zerodev"` that we could use instead of what's now a bit of a magic string

Reply:
It's a `simpleWebAuthn/browser` error, but they don't export the error codes.

speaking of which, neither do we, I'll add that in another PR.